Makefile

```diff
@@ -1,3 +1,11 @@
+install:
```

Comment:
Maybe make `dep` a dependency of `install` and update the readme to just call that?